carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/impl/UIAutomator2Capabilities.java

```diff
@@ -0,0 +1,85 @@
+package com.qaprosoft.carina.core.foundation.webdriver.core.capability.impl;
+
+import java.lang.invoke.MethodHandles;
+
+import org.openqa.selenium.Capabilities;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.qaprosoft.carina.core.foundation.utils.Configuration;
+import com.qaprosoft.carina.core.foundation.utils.R;
+import com.qaprosoft.carina.core.foundation.webdriver.core.capability.AbstactCapabilities;
+
+import io.appium.java_client.android.options.UiAutomator2Options;
+
+public class UIAutomator2Capabilities extends AbstactCapabilities<UiAutomator2Options> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @Override
+    public UiAutomator2Options getCapabilities() {
+        UiAutomator2Options options = new UiAutomator2Options();
+        // this step should be executed before getConfigurationCapabilities() to be able to override this capabilities by default appium approach.
+        setLocaleAndLanguage(options);
+        setCapabilitiesSafe(options, getConfigurationCapabilities());
+        return options;
+    }
+
+    @Override
+    public UiAutomator2Options createCapabilitiesFromCustom(Capabilities customCapabilities) {
```

Comment:
let's think about better naming for "withCustom" andd "fromCustom"

Reply:
This method removed